src/Microsoft.TestPlatform.ObjectModel/Utilities/XmlRunSettingsUtilities.cs

```diff
@@ -26,6 +26,29 @@ public static ObjectModel.Architecture OSArchitecture
         {
             get
             {
+#if !NETSTANDARD1_4
```

Comment:
Can't this come from the PlatformEnvironment ?

Reply:
No that enum doesn't have ARM63 in it

Reply:
No due to backwards compat reasons this API normally lies about the OS architecture, so this is the way to do it.